tox.ini

```diff
@@ -1,4 +1,5 @@
 [tox]
+isolated_build = true
```

Comment:
why is this line necessary now?

Reply:
what was it responsible for originally? not sure i understood [the documentation](https://tox.wiki/en/3.3.0/config.html#confval-isolated_build=True|False(default))

It creates a new fresh environment to install the package and test it but now poetry does this for us so keeping it would generate a fresh env inside a fresh env, did I get it right @mateuslatrova ?